amd/hipcc/docs/index.rst

```diff
@@ -0,0 +1,34 @@
+.. meta::
+  :description: HIPCC command
+  :keywords: HIPCC, ROCm, HIP tools, HIP compiler
+
+.. _hipcc-docs:
+
+******************************************
+HIPCC documentation
+******************************************
+
+``hipcc`` is a compiler driver utility that will call clang or nvcc, depending on target, and pass the appropriate include and library options for the target compiler and HIP infrastructure. 
+
+There is both a Perl version, and a C++ executable version of the ``hipcc`` and ``hipconfig`` compiler driver utilities provided. By default the Perl version is used when ``hipcc`` is run. To enable the C++ versions, set the environment variable ``HIP_USE_PERL_SCRIPTS=0``.
+
```

Comment:
For ROCm 6.2, hipcc and hipconfig will default to use the C++ versions instead of the Perl versions - the opposite of what we have here and what was done in previous releases.  So, can we change the wording to something like this:

In ROCm 6.2, the C++ executables "hipcc.bin" and "hipconfig.bin" will be invoked by default whenever "hipcc" or "hipconfig" are called, respectively.   To have the Perl versions called instead, set the environment variable "HIP_USE_PERL_SCRIPTS=1" before calling "hipcc" or "hipconfig". 

The Perl versions of the scripts are deprecated, and will be removed in a future ROCm release.

Reply:
If you want, we can use this PR in 6.1.1, and I can change it for 6.2 as you have indicated. If you just want to use this for 6.2 I will change it as requested.

Reply:
This change, in default behaviour, is only for ROCm 6.2.

Reply:
ok, so we can use this change (doc restructuring) as-is for ROCm 6.1.1.  And change the wording for ROCm 6.2.